integration/spring-pulsar/src/main/java/com/example/pulsar/SpringPulsarApplication.java

```diff
@@ -0,0 +1,42 @@
+package com.example.pulsar;
+
+import org.apache.pulsar.client.api.Schema;
+import org.apache.pulsar.common.schema.SchemaType;
+
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.pulsar.annotation.PulsarListener;
+import org.springframework.pulsar.core.PulsarTemplate;
+
+@SpringBootApplication
+public class SpringPulsarApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(SpringPulsarApplication.class, args);
+    }
+
+    @Bean
+    ApplicationRunner runner1(PulsarTemplate<Greeting> pulsarTemplate) {
+        String topic = "graal-topic";
+        pulsarTemplate.setSchema(Schema.JSON(Greeting.class));
+        return args -> {
+            pulsarTemplate.sendAsync(topic, new Greeting("Hello from GraalVM!"));
+        };
+    }
+
+    @PulsarListener(subscriptionName = "graalvm-demo-subscription", topics = "graal-topic",
+            schemaType = SchemaType.JSON)
+    void process(Greeting message) {
```

Comment:
**[TinyNit]** It may help w/ readability if we name the method something more descriptive along the lines of what it does, maybe something like `receiveMessageFromTopic()`. It seems nit-picky but if someone looks in here to debug something later and they know nothing about spring-pulsar, this could make it easier to follow.